### DISCUSSION TITLE
ESP32 + ILI9481_DRIVER doesnt seem to be working

### DISCUSSION THREAD

**Discussion by Bharat-Pi** [Bodmer/TFT_eSPI#2887] [Q&A - General]
**Using TSP_eSPI version:** 2.5.0

Tried both #define ILI9481_DRIVER and 

Below is the out put of setup program showing my pin connections from ESP32 to Display. Getting the below mentioned error when running the read write test program. What could be issue?

**Read User Setup program output:**
[code]
TFT_eSPI ver = 2.5.0
Processor    = ESP50
Frequency    = 240 MHz
Transactions = Yes 
Interface    = Parallel 
Display driver = 9481
Display width  = 320 
Display height = 480 

TFT_CS   = D4 (GPIO 2)
TFT_DC   = D6 (GPIO 12)
TFT_RST  = D2 (GPIO 4)

TFT_WR   = D7 (GPIO 13)
TFT_RD   = D5 (GPIO 14)

TFT_D0   = D12 (GPIO 18)
TFT_D1   = D12 (GPIO 19)
TFT_D2   = D0 (GPIO 16)
TFT_D3   = D12 (GPIO 17)
TFT_D4   = D1 (GPIO 5)
TFT_D5   = D12 (GPIO 25)
TFT_D6   = D12 (GPIO 26)
TFT_D7   = D12 (GPIO 27)

Font GLCD   loaded
Font 2      loaded
Font 4      loaded
Font 6      loaded
Font 7      loaded
Font 8      loaded
Smooth font enabled

[/code]

**TFT Read write test program out put:**
 Pixel value read    = 900
 ERROR                 ^^^^
 Pixel value written = 80
 Pixel value read    = 900
 ERROR                 ^^^^
 Pixel value written = 100
 Pixel value read    = 900
 ERROR                 ^^^^
 Pixel value written = 200
.....

Attacheding image of the setup:
![IMG_8341](https://github.com/Bodmer/TFT_eSPI/assets/133974527/b004d168-2a1f-4128-ba8b-03e125e00e21)

![IMG_8342](https://github.com/Bodmer/TFT_eSPI/assets/133974527/a55d8956-4ce2-4908-87d1-d117960de234)

![IMG_8343](https://github.com/Bodmer/TFT_eSPI/assets/133974527/d1325ef9-9e85-4fa8-9887-fdc248e1060f)

**Comment by Bodmer** [comment]
Try other examples that do not read the display to make sure the interface is connected and configured correctly.
